.babelrc

```diff
@@ -17,10 +17,39 @@
         "debug": true,
         "modules": false,
         "targets": {
-          "browsers": ["chrome >= 56", "firefox >= 52", "edge >= 14", "opera >= 43"]
+          "browsers": [
+            "chrome >= 56",
+            "firefox >= 52",
+            "edge >= 14",
+            "opera >= 43"
+          ]
         },
         "useBuiltIns": true
       }
     ]
-  ]
+  ],
+  "env": {
+    "test": {
+      "presets": [
+        "jest",
+        "react",
+        [
+          "env",
+          {
+            "debug": false,
+            "modules": "commonjs",
+            "targets": {
```

Comment:
Targets is duplicated now. Is it possible to extract that?

Reply:
It overrides the first target.